backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/database/entity/AccessRecordId.kt

```diff
@@ -0,0 +1,10 @@
+package com.tngtech.apicenter.backend.connector.database.entity
+
+import java.io.Serializable
+import javax.persistence.Embeddable
+
+@Embeddable
+data class AccessRecordId(
```

Comment:
That isn't possible without a new class?

Reply:
My interpretation of the type parametrisation of a repository was that it requires an ID class? eg. `interface MyRepository<T, ID extends Serializable>
  extends JpaRepository<T, ID>`

Reply:
That's unfortunate, but c'est la vie :p